docs/reference/commandline/container_run.md

```diff
@@ -1283,7 +1283,7 @@ connect to services running on the host machine.
 
 It's conventional to use `host.docker.internal` as the hostname referring to
 `host-gateway`. Docker Desktop automatically resolves this hostname, see
-[Explore networking features](https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host).
+[Explore networking features](https://docs.docker.com/desktop/features/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host).
```

Comment:
The new URL is currently a 404, is that expected? https://docs.docker.com/desktop/features/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host

Reply:
yes because it's updated by the linked PR

Reply:
OH! It's not merged yet 😅 

Let's wait with merging until that one is in (I see redirects are added as well, so old one should still work, correct?)

Reply:
No we should merge this first, because otherwise CI would be failing https://github.com/docker/docs/actions/runs/11702658384/job/32591295114?pr=21354#step:4:418

Reply:
OH! CI doesn't allow for redirects? 😢